src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -187,7 +187,7 @@ function informationDiagram(publicAPI, model) {
 
     if (variableList.length < 2 || !model.container) {
       // Select the main circle and hide it and unhide placeholder
-      d3.select(model.container).select('svg.information-diagram').classed(style.hidden, true);
+      d3.select(model.container).select('svg.information-diagram').attr('class', style.informationDiagramSvgHide);
```

Comment:
Unfortunately MI is using a mix of direct class names and CSS Modules, and they encourage different coding styles. If I have the time I'll try to convert to pure CSS Modules style, like HistogramSelector